### ISSUE TITLE
Optimistic updates and unfinished api calls

### ISSUE THREAD

**Issue by gabrielgiussi** [acdlite/redux-promise#17]
I already posted in [stackoverflow] (http://stackoverflow.com/q/31887573/3517383) about this because i don't know what to do when the user closes the page or the session and there is pending actions that has been showed to the user like finished (example: a created todo item), after all this is what optimistic updates are.

If you see the code posted the Action object doesn't matter anymore because it should be replaced by redux-promises, but the need of querying for pending actions still exists.

Basically the example consist on an ActionReducer (in fact it says ActionStore because was created previous to redux 1.0) that handles the state (pending | finished) of all the actions in the app. This makes easy to query if there are pending actions before the user closes the app. 

It makes sense what I'm asking? The action store is a feasible approach?
